app/src/main/java/com/example/faunaapp/DTO/Entry.java

```diff
@@ -1,12 +1,29 @@
 package com.example.faunaapp.DTO;
 
 public class Entry {
```

Comment:
Try and be more expressive in your naming. Is this a TaskEntry?

Reply:
yeah